website/docs/cdktf/cli-reference/commands.html.md

```diff
@@ -380,39 +388,42 @@ Synthesize code when providing a custom command to execute and an output directo
 $ cdktf synth --app="npm compile && node main.js" --output="dirname"
 ```
 
-Synthesize code for a given application and output the Terraform JSON configuration.
+Synthesize code for an application and output the Terraform JSON configuration.
 
 ```bash
 $ cdktf synth --json
 ```
 
 ## watch
 
-~> **Warning:** The `watch` command is experimental. You should use it in development environments.
+~> **Warning:** The `watch` command is experimental, so you should only use it in development environments. It also automatically deploys all changes without asking for confirmation.
 
 The `watch` command watches a directory for changes and automatically synthesizes and deploys changes as they happen. It allows for rapid iterations when developing infrastructure, especially when working with serverless services. It currently supports only one stack at a time.
 
 Watch reads your root `.gitignore` file to determine which files trigger a `synth` and it uses the `outdir` of your `cdktf.json` (or the default `cdktf.out`) to do so for a deploy. It will generate a checksum of the subdirectory containing the Terraform code for your stack to skip deploys if the synthesized Terraform config did not change.
 
-### Checking your root `.gitignore`
 
-Your root `.gitignore` needs to contain all files that are generated by a synth and a deploy. If that is not the case the watch will trigger itself and continuously try to deploy. Although Git supports nesting `.gitignore` files, the watch command currently only reads the root `.gitignore` file.
+### Prerequisites
+
+Before using `watch` you should:
+
+- **Check your root `.gitignore` file.** Your root `.gitignore` must contain all files that are generated by `synth` and `deploy`. If that is not the case, the `watch` command will trigger itself and continuously try to deploy. Although Git supports nesting `.gitignore` files, the `watch` command currently only reads the root `.gitignore` file.
 
-### Checking your environment
+- **Check your environment.** The `watch` command should only be used for development environments. We recommend making sure that the terminal where you want to run `watch` has no access keys that allow the `cdktf-cli` to deploy to your production environment.
 
-Watch should only be used for development environments. It is best to make sure that the terminal you're running watch in has no access keys that allow the cdktf-cli to deploy to your production environment.
+### Run
 
-### Running watch
+An example use case for watch could be:
 
-An exemplary invocation of watch could be:
+TODO please provide an explanation of what these flags do!
 
 ```
 cdktf watch --stack dev --auto-approve
 ```
```

Comment:
I think `--stack` is just a positional argument according to the help output. `auto-approve` behaves similar to TF Core and will apply changes without additional confirmation. `auto-approve` is mandatory right now.

```
cdktf watch [stack] [OPTIONS]

[experimental] Watch for file changes and automatically trigger a deploy

Positionals:
  stack  Deploy stack which matches the given id only. Required when more than one stack is present in the app                                                                                                                                                                                                                                                      [string]

Options:
      --version                   Show version number                                                                                                                                                                                                                                                                                                              [boolean]
      --disable-logging           Dont write log files. Supported using the env CDKTF_DISABLE_LOGGING.                                                                                                                                                                                                                                             [boolean] [default: true]
      --disable-plugin-cache-env  Dont set TF_PLUGIN_CACHE_DIR automatically. This is useful when the plugin cache is configured differently. Supported using the env CDKTF_DISABLE_PLUGIN_CACHE_ENV.                                                                                                                                             [boolean] [default: false]
      --log-level                 Which log level should be written. Only supported via setting the env CDKTF_LOG_LEVEL                                                                                                                                                                                                                                             [string]
  -a, --app                       Command to use in order to execute cdktf app                                                                                                                                                                                                                                                                                    [required]
  -o, --output                    Output directory                                                                                                                                                                                                                                                                                         [required] [default: "cdktf.out"]
      --auto-approve              Auto approve                                                                                                                                                                                                                                                                                                    [boolean] [default: false]
  -h, --help                      Show help
```

Reply:
okay! I added in this help text, added in an explanation for the example, and fussed with the formatting a bit to make it match the other sections on the page.  I'm gonna push and we can make any final tweaks in the pass I want to make before Hashiconf :-)